xsl/assembly/effectivity.xsl

```diff
@@ -0,0 +1,113 @@
+<?xml version="1.0" encoding="utf-8"?>
+<xsl:stylesheet 
+  xmlns:xsl="http://www.w3.org/1999/XSL/Transform" 
+  xmlns:d="http://docbook.org/ns/docbook"
+  xmlns:exsl="http://exslt.org/common"
+  xmlns:xlink="http://www.w3.org/1999/xlink"
+  xmlns="http://docbook.org/ns/docbook"
+  exclude-result-prefixes="exsl d xlink d"
+  version="1.0">
+  
+  <xsl:param name="effectivity.arch" />
+  <xsl:param name="effectivity.audience" />
+  <xsl:param name="effectivity.condition" />
+  <xsl:param name="effectivity.conformance" />
+  <xsl:param name="effectivity.os" />
+  <xsl:param name="effectivity.outputformat" />
+  <xsl:param name="effectivity.revision" />
+  <xsl:param name="effectivity.security" />
+  <xsl:param name="effectivity.userlevel" />
+  <xsl:param name="effectivity.vendor" />
+  <xsl:param name="effectivity.wordsize" />
+
+  <xsl:template match="d:module[@resourceref] | d:structure[@resourceref]" mode="evaluate.effectivity">
+    <xsl:apply-templates mode="evaluate.effectivity" />
+  </xsl:template>
+  
+  <xsl:template match="d:filterout" mode="evaluate.effectivity">
```

Comment:
@nvijaybhaskar-toast, I think this template handles the `filterout` element correctly now.